oidc-controller/api/authSessions/models.py

```diff
@@ -27,6 +27,7 @@ class AuthSessionBase(BaseModel):
     pyop_auth_code: str
     response_url: str
     presentation_request_msg: dict | None = None
+    socket_id: str | None = None
```

Comment:
Think we should index this field since there's a `get by` I think, and be good to add a unique constraint? 

I don't think we'd be adding an SID to more than one auth session record right? Or I could be wrong about that and we create new auth sessions that keep the SID... if not I'd do a unique constraint (would notice pretty quick if it breaks)

Reply:
Yeah trying out locally think we can go with a unique constraint on the SID + the index